serverless/monitoring-serverless.adoc

```diff
@@ -0,0 +1,11 @@
+[id="monitoring-serverless"]
+= Monitoring Serverless applications
+include::modules/common-attributes.adoc[]
+:context: cluster-monitoring-serverless
+
+As an {product-title} cluster administrator, you can deploy the {product-title} monitoring stack and monitor the status of each deployment as well as metrics data.
+
+When using the OpenShift Serverless operator, the required ServiceMonitor objects are created automatically for cluster monitoring on Serverless applications. OpenShift Serverless components, such as Knative Serving, expose metrics data, and administrators can monitor this data by using the {product-name} Web console.
```

Comment:
You only seem to use `product-name` here? Should it be `product-title`?

Reply:
Thanks, it's a typo I guess, will update.